src/wp-includes/class-wp-navigation-fallback.php

```diff
@@ -26,9 +26,18 @@ class WP_Navigation_Fallback {
 	 */
 	public static function get_fallback() {
 
+		/**
+		 * Filters whether or not a fallback should be created.
+		 *
+		 * @since 6.3.0
+		 *
+		 * @param bool Whether or not to create a fallback.
+		 */
+		$should_create_fallback = apply_filters( 'wp_navigation_should_create_fallback', true );
```

Comment:
I'm unsure as to whether the `wp_` prefix is correct.

In the original Gutenberg PR this was prefixed with `gutenberg_`.

Reply:
The `wp_` prefix is good. 👍🏻 

`gutenberg_*` shouldn't be in core files.

Any third parties who want to use this filter will forever be using `wp_navigation_should_create_fallback` 😄